src/kernel/sporadic.c

```diff
@@ -143,7 +143,7 @@ static inline void maybe_add_empty_tail(sched_context_t *sc)
 {
     if (isRoundRobin(sc)) {
         /* add an empty refill - we track the used up time here */
-        refill_t empty_tail = { .rTime = NODE_STATE(ksCurTime)};
+        refill_t empty_tail = {0};
```

Comment:
```suggestion
        refill_t empty_tail = { .rTime = refill_head(sc)->rTime };
```

Reply:
Testing this change now.

Reply:
This has been changed as per suggestion.